src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -80,7 +82,7 @@
               (let [pairs (core/partition 2 v1)
                     names (map first pairs)
                     gens (map second pairs)]
-                `(for [[~@names] (gen/tuple ~@gens)
+                `(for [[~@names] (~'clojure.test.check.generators/tuple ~@gens)
```

Comment:
Hummm, not sure why we left it that way... Perhaps we needed something like that when clojure.test.check had separate namespaces for clj & cljs, but it's not the case anymore. Just changed it to use the `gen` alias